src/Item/Item.router.ts

```diff
@@ -141,9 +142,11 @@ export class ItemRouter extends Router {
       Item.find<ItemAttributes>(),
       collectionAPI.fetchItems(),
     ])
-    const catalystItems = await peerAPI.fetchWearables(
-      remoteItems.map((item) => item.urn)
-    )
+
+    const catalystItems =
+      remoteItems && remoteItems.length > 0
+        ? await peerAPI.fetchWearables(remoteItems.map((item) => item.urn))
+        : []
```

Comment:
Why's this necessary? can `remoteItems` be `undefined`? the type says `Promise<ItemFragment[]>`
peer.ts already has this check:
```ts
const wearables: PeerWearable[] =
  urns.length > 0
    ? await this.lambdasClient.fetchWearables({ wearableIds: urns })
    : []
```

same thing [here](https://github.com/decentraland/builder-server/pull/286/files#diff-5db28707b1b7436bf2151ad65251263f4b79eeddc180a1a22996464abd5866c4R181)

Reply:
You're right, there was no need to check for undefined. Removed it in both places.